ddd-koans/Entity/Lifecycle/task.html

```diff
@@ -0,0 +1,27 @@
+<html>
+<h2>Lifecycle</h2>
+<p>
+    In the beginning, while creating a piece of music, bars don't exist in the piece at all.
+    Then creators add bars one after another. They decide what time signature should
+    be used and what notes should fill the bar.
+</p>
+<p>
+    As the definition of the bar states, there is a limit on notes in the bar. So creators
+    can add notes just to fill the limit. But, the bar is only complete when
+    is filled with enough notes to match the beats corresponding to the bar's time signature.
+</p>
+<p>
+    To recall information about time signature in a bar:
+<ul>
+    <li>The note value of the time signature defines which note value is equal to one beat.</li>
+    <li>The number of beats declares how many beats makes a complete bar.</li>
+</ul>
+    Therefore, to create a complete bar, the sum of beats of note values in the bar must be equal
+    to the time signature's number of beats.
+</p>
+<p>
+    To sum up, a bar can not yet exist, be in an incomplete state or be in a complete state.
```

Comment:
I don't quite get it, I can propose below narrative to reflect a little langue and the example.

```
Hey, man! You need to complete those bars in our music. They are incomplete. Please put there your drum section - said bassist

Uh, yeah, I will put there missing notes - responded drummer

```